toolchain/check/pattern_match.h

```diff
@@ -10,15 +10,14 @@
 
 namespace Carbon::Check {
 
-// The pattern-match counterparts of the patterns passed to CalleePatternMatch.
+// The outputs of CalleePatternMatch.
+// TODO: Rename or remove this struct.
 struct ParameterBlocks {
-  // The implicit parameter list.
-  SemIR::InstBlockId implicit_params_id;
-
-  // The explicit parameter list.
-  SemIR::InstBlockId params_id;
+  // The call parameters of the function.
```

Comment:
Sorry, I missed this last time.
```suggestion
  // The `Call` parameters of the function.
```